wellcomeml/ml/spacy_classifier.py

```diff
@@ -28,7 +28,7 @@ class SpacyClassifier(BaseEstimator, ClassifierMixin):
     def __init__(self, threshold=0.5, n_iterations=5,
                  batch_size=8, learning_rate=0.001,
                  dropout=0.1, shuffle=True, architecture="simple_cnn",
-                 pre_trained_vectors_path=None):
+                 exclusive_classes=False, pre_trained_vectors_path=None):
```

Comment:
:books: Perhaps some doctoring here will help, if you want. I had to read the PR explanation to understand what exclusive_classes does.